### ISSUE TITLE
Potential tooltip for month field to highlight Quarters

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeBMD2#405]

**Comment by PatReynolds** [comment]
@PatReynolds  @AlOneill and @DeniseColbert to discuss wording of words on the search form and placement on Friday

**Comment by DeniseColbert** [comment]
@PatReynolds to test dropdown in screen reader software

**Comment by DeniseColbert** [comment]
@Vino-S has changed the label of Q & Year which might help...

**Comment by AlOneill** [comment]
@Vino-S The forward slash does not have much meaning (in written English) apart from in and/or. Also, a screen-reader user will likely not be made aware that it is there.

Suggest it is replaced with " and ".

However, I'm not sure whether or not a screen-reader user will understand what is going on. The visible label is tied to the quarter input but it mentions quarter _and_ year; the (invisible to most users) label ("From year" is tied to the year input. So, a screen-reader user could be expecting to select both quarter and year in the first input (if you get what I mean!).

I'm not fluent in Aria, but think that we can use it to hide part of the visible label from a screen-reader user. Will investigate …

**Comment by richpomfret** [comment]
@AlOneill any update on this one?

**Comment by AlOneill** [comment]
Apologies — I had forgotten about this.

I've a feeling that the markup for the 4 date inputs is not quite right for SR users. I am taking a closer look with the hope of offering a concrete solution.

And any tooltip would have to work for all our users. Might be tricky because of what we are trying to hide from SR users versus what they need to have access to.

**Comment by AlOneill** [comment]
What do we want the tooltip to say? What should it explain? Space will be limited (at some window widths), so the text will have to be concise. 

Irrespective of whether or not we provide a tooltip, the current markup for the date fields is not accessible to SR users — they are misled about the "Quarter" drop-downs (the select boxes). The labels for the "Quarter" selects are "From Quarter and Year" and "To Quarter and Year", which is not correct.

The fix would be straightforward without including a tooltip, but with one, the solution is a little more involved because we cannot hide focusable elements — the tooltip — from SR users. Also the Quarter fields by themselves are not wide enough (in a wide window) to hold a tooltip longer than a handful of words.

I am in the process of testing a possible solution for the with-tooltip option.

**Comment by DeniseColbert** [comment]
Decision: Tooltip to contain WTTE "See Help for more info"

**Comment by AlOneill** [comment]
Whilst exploring this issue, I became aware that our current markup for tooltips is not fully accessible — mea culpa. We should attach the tooltip to the input rather than the label: this will require a change in markup and CSS. I'm not going to rush this as it should be fixed once and for all.

In the meantime, do we want an improved (but not correct or fully accessible) temporary fix?

Once I'm clear on the exact requirements of a truly accessible tooltip, I'll open an issue on System-wide.

**Comment by PatReynolds** [comment]
Agreed, @AlOneill to impplement a temporary fix

**Comment by PatReynolds** [comment]
No easy fix, so placing on ice until later

**Comment by KristinaGadzhieva** [comment]
No longer relevant. A link to the help page will be added on the search page.
